corehq/apps/locations/views.py

```diff
@@ -164,7 +164,10 @@ def mk_loctype(name, parent_type, administrative,
             if loc_type is None:
                 loc_type = LocationType(domain=self.domain)
             loc_type.name = name
-            loc_type.administrative = administrative
+            # only CommTrack domains should have non-administrative locations
+            loc_type.administrative = (administrative
```

Comment:
what is being passed in as `administrative` here? it kind of seems like this logic should go wherever that variable is set, since it'd be pretty confusing to pass in `administrative=False` and then still end up with it being set for some reason inside the function you don't expect